### PR TITLE
add micormenter

### PR DESCRIPTION
**Pull request by arufanov**
Micrometer has been added

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,7 +64,15 @@
...
             <optional>true</optional>
         </dependency>
-
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-actuator</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.micrometer</groupId>
+            <artifactId>micrometer-registry-prometheus</artifactId>
```
Comment (yurichervoniy) [inline pom.xml:73]: Why do we need that? 
You can look in starter-actuator - it already has dependecy

Comment (arufanov) [reply pom.xml:73]: @yurichervoniy 
* it doesn't have micrometer-registry-prometheus

**Review by yurichervoniy — APPROVED** [review]

**Comment by arufanov** [comment]
@yurichervoniy 
* could you review it? I am not able to find actuator at startup log.

**Comment by arufanov** [comment]
@yurichervoniy 
* this mr doesn't work
* could you review it? I am not able to find actuator at startup log.

**Review by yurichervoniy — APPROVED** [review]

**Pull request merged**